CONTRIBUTING.md

```diff
@@ -17,30 +17,7 @@ By contributing your code, you agree to license your contribution under the term
 
 All files are released with the Apache 2.0 licence.
 
-If you are adding a new file it should have a header comment containing licence information:
-
-<details>
-<summary>Show/hide licence header</summary>
-
-```
-Backpack - Skyscanner's Design System
-
-Copyright 2016-2021 Skyscanner Ltd
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-```
-
-</details>
+If you are adding a new file it should have a header comment containing licence information from the following file: [license](./license.txt).
```

Comment:
Changed this in favour to linking to the license file as its the exact same but just pasted here, so ensures we only need to update one place and can't miss it here